### DISCUSSION TITLE
I have a question about the sendToDeviceGroup function.

### DISCUSSION THREAD

**Discussion by dev-tryit** [firebase/firebase-admin-node#1510] [Q&A]
I have a question about the sendToDeviceGroup function.

sendToDeviceGroup has three parameters.
`sendToDeviceGroup(notificationKey: string, payload: MessagingPayload, options?: MessagingOptions): Promise<MessagingDeviceGroupResponse>;`

Among these, I am curious about MessagingOptions.

In MessagingOptions, the priority field defaults to high.
(https://firebase.google.com/docs/reference/admin/node/firebase-admin.messaging.messagingoptions.md#messagingoptions_interface)

if MessagingOptions is undefiened, Is the message priority set high when message notifications are sent?

**Comment by lahirumaramba** [comment]
Looking at the reference docs and the [REST API ](https://firebase.google.com/docs/cloud-messaging/http-server-ref#send-downstream), if `MessagingOptions` are not defined the API will use the default priorities, which is `normal` for data messages and `high` for notification messages.

Adding @chong-shao to confirm this as `sendToDeviceGroup` uses the FCM legacy API.
